assets/tests/components/routeLadders.test.tsx

```diff
@@ -115,22 +116,8 @@ describe("RouteLadders", () => {
         .mockImplementationOnce(() => timepointsByRouteId)
       jest.mocked(useActiveDetoursByRoute).mockReturnValue({
         "1": {
-          "1": {
-            id: 1,
-            route: "1",
-            direction: "Inbound",
-            name: "",
-            intersection: "A St & B Av",
-            updatedAt: 1724866392,
-          },
-          "2": {
-            id: 2,
-            route: "1",
-            direction: "Outbound",
-            name: "",
-            intersection: "C Rd & D Ct",
-            updatedAt: 1724866392,
-          },
+          "1": simpleDetourFactory.build(),
+          "2": simpleDetourFactory.build(),
```

Comment:
(Context https://github.com/mbta/skate/pull/2915#discussion_r1923881801)
Thanks for the factory changes!!

I think you need to keep `id: <value>` in these, to keep the data correct, since the id of the `simpleDetour` should match the id of the key in the map, e.g.,

```suggestion
          "1": simpleDetourFactory.build({id: 1}),
          "2": simpleDetourFactory.build({id: 2}),
```

but otherwise looks great 👍🏻

Reply:
This change would need to be made in other files too but I _think_ it'd mostly be fine if we ran into that later?

Reply:
Ah I see, that's fair! I omitted it since we're only checking for visibility, but great catch that it's not totally accurate. I'll make the change here and in the one other place I added the factory